services/rapid-analysis/values-usdf-comcamsim.yaml

```diff
@@ -0,0 +1,123 @@
+namespace: rubintv-comcamsim
+image:
+  repository: lsstts/rapid-analysis
+  tag: c0035_usdf
```

Comment:
There is a more recent tag for the `lsstts/rapid-analysis` image you could use: [c0037_usdf](https://hub.docker.com/layers/lsstts/rapid-analysis/c0037_usdf/images/sha256-27522a49624ca39da6fedec0e529c4d4a7dcaf84c8140da3cad9ae8487da7b83?context=explore).

Reply:
Updated to c0037_usdf.